task5/src/main/java/com/example/demo111/Check.java

```diff
@@ -0,0 +1,30 @@
+package com.example.demo111;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(name = "check", value = "/check")
+public class Check extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        doPost(request, response);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        try {
+
+            response.setContentType("json");
+            PrintWriter out = response.getWriter();
+            out.print("{\"success\":true}");
+            out.flush();
+        } catch (Exception e) {
+
```

Comment:
Please do never leave empty catch blocks, this is bad practice
Also, please note that your function `throws IOException`. That means `IOException` should not bother for the in-method code. But `catch (Exception ex)` covers all the exceptions, including IOException itself. So, it will never be thrown.